source/php/Component/Tags/Tags.php

```diff
@@ -17,6 +17,23 @@ public function init()
             $this->data['classList'][] = $this->getBaseClass() . "--format";
         }
 
+        
+        if (function_exists('get_theme_mod')) {
```

Comment:
This is not allowed here. Use Municipio context parameter to sent attribute or modifiers to component.

Reply:
Same for below settings.